### PR TITLE
Fixed and added new test for documents.

### PR DESCRIPTION
**Pull request by mindaugasrukas**

### CODE REVIEW

File: Document/CategoryDocument.php [hunk 1]
```diff
@@ -85,6 +87,16 @@ public function addExpiredUrl($expiredUrl)
     }
 
     /**
+     * Remove expired url from array.
+     *
+     * @param string $expiredUrl
+     */
+    public function removeExpiredUrl($expiredUrl)
```
Comment (tautrimas) [inline Document/CategoryDocument.php:94]: This should be located in [SeoAwareTrait](https://github.com/ongr-io/RouterBundle/blob/master/Document/SeoAwareTrait.php).

Also, `$urls` and `$expiredUrls` properties are defined. Shouldn't they be included using that SeoAwareTrait?

Comment (mindaugasrukas) [inline Document/CategoryDocument.php:94]: working on it..

Comment (mindaugasrukas) [inline Document/CategoryDocument.php:94]: depends on https://github.com/ongr-io/RouterBundle/issues/5

**Comment by tautrimas** [comment]
This is a conflict with #25.

**Comment by tfe2012** [comment]
I think it's not be actual after https://github.com/ongr-io/MagentoConnectorBundle/pull/48 commit

**Pull request closed without merging**